TestCases/sliding_interface/bars_SST_2D/bars.cfg

```diff
@@ -252,7 +252,7 @@ SLOPE_LIMITER_TURB= NONE
 TIME_DISCRE_TURB= EULER_IMPLICIT
 %
 % Reduction factor of the CFL coefficient in the turbulence problem
-CFL_REDUCTION_TURB= 0.10
+CFL_REDUCTION_TURB= 1.0
```

Comment:
I changes all but 2 test cases to use CFL_REDUCTION_TURB=1 (the lower value previously had no effect and so the residuals changed).